modules/services/osm.js

```diff
@@ -1056,11 +1063,15 @@ export default {
 
         var that = this;
         var path = '/api/0.6/notes?limit=' + noteOptions.limit + '&closed=' + noteOptions.closed + '&bbox=';
-        var throttleLoadUsers = _throttle(function() {
-            var uids = Object.keys(_userCache.toLoad);
-            if (!uids.length) return;
-            that.loadUsers(uids, function() {});  // eagerly load user details
-        }, 750);
+
+        // throttle to avoid unncessary API calls
+        if (!this.throttledLoadUsers) {
+            this.throttledLoadUsers = _throttle(function() {
+                var uids = Object.keys(_userCache.toLoad);
+                if (!uids.length) return;
+                that.loadUsers(uids, function() {});  // eagerly load user details
+            }, 1000);
+        }
```

Comment:
Good question!  this comes from 6424e8501 which was me trying to get the tests working again.

We use [lodash.throttle](https://lodash.com/docs/4.17.15#throttle) in two places in the osm service - fetching user data attached to notes, and checking OSM API status.

In both cases, we really need to store this throttled call somewhere, and cancel it whenever we reset the osm service.  The throttling was happening before, it just wasn't stored in `this`, nor was it cancelled.  This could introduce a problem when switching between the prod and dev osm server, (e.g. we switch to dev but then end up fetching the prod api status).

I noticed when running the tests that the throttled calls to check for the API status would persist between tests, introducing a bunch of failures.  (e.g. it would perform a mocked API call like loadTiles, but then some milliseconds later attempt the API status call).

So now I'm trying to cancel them every time we call `reset`, which happens between each test, so these throttled calls shouldn't interfere with any tests.